presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -217,3 +217,19 @@ Unicode Functions
     are replaced with ``replace``. The replacement string ``replace`` must either
     be a single character or empty (in which case invalid characters are
     removed).
+
+.. function:: to_charset(charset, string) -> varbinary
+
+    Encodes ``string`` into a ``charset`` varbinary representation.
+
+.. function:: from_charset(charset, binary) -> varchar
+
+    Decodes a ``charset`` encoded string from ``binary``. Invalid sequences
```

Comment:
I've been looking at character set conversion, and it's not clear that `U+FFFD` is always the replacement character. It is for our UTF-8 implementation, and for many character sets, but the Javadoc for `String` says this:

> This method always replaces malformed-input and unmappable-character sequences with this charset's default replacement string.

I'm not sure if this is a practical concern, but I'd rather not make any such guarantees in the documentation, as I'm not an expert in this area. Given this, I think we should change this sentence:

```
Invalid sequences are replaced depending on the character set, typically with the Unicode replacement character ``U+FFFD``.